protocol/messenger_communities.go

```diff
@@ -5155,3 +5157,28 @@ func (m *Messenger) startRequestMissingCommunityChannelsHRKeysLoop() {
 		}
 	}()
 }
+
+// getCommunityMailserver returns the active mailserver if a communityID is present then it'll return the mailserver
+// for that community if it has a mailserver setup otherwise it'll return the global mailserver
+func (m *Messenger) getCommunityMailserver(communityID ...string) peer.ID {
```

Comment:
At least for the new functions 😄 
```suggestion
func (m *Messenger) getCommunityStorenode(communityID ...string) peer.ID {
```